hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SequenceIdGenerator.java

```diff
@@ -177,7 +181,7 @@ public void reinitialize(Table<String, Long> sequenceIdTable)
     lock.lock();
     try {
       LOG.info("reinitialize SequenceIdGenerator.");
```

Comment:
I think we should move the log statement out of the lock.

Reply:
sure.